tests/src/FunctionalJavascript/Form/ImporterFormTest.php

```diff
@@ -0,0 +1,205 @@
+    $data_fetcher_urls->setValue('http://news.com/api/articles');
+
+    // Assert data parser configuration matches up with our migration.
+    $data_parser_tab = $page->find('css', '[href="#plugin_settings--data_parser"]');
+    $data_parser_tab->click();
+    $data_parser_plugin = $page->findField('migration[source][data_parser_plugin]');
+    $this->assertTrue($migration->get('source')['data_parser_plugin'], $data_parser_plugin->getValue(), 'Data parser plugin matches with migration.');
+
+    // Assert destination configuration matches up with our migration.
+    $destination_tab = $page->find('css', '[href="#plugin_settings--destination"]');
+    $destination_tab->click();
+    $destination_plugin = $page->findField('migration[destination][plugin]');
+    $this->assertTrue($migration->get('destination')['plugin'], $destination_plugin->getValue(), 'Destination plugin matches with migration.');
+    $destination_bundle = $page->findField('destination_wrapper[options][default_bundle]');
+    $this->assertTrue($migration->get('destination')['default_bundle'], $destination_bundle->getValue(), 'Destination bundle matches with migration.');
+
+    // Submit the form.
+    $this->submitForm([], 'Save');
+
+    // Check if importer is saved with the expected values.
+    $importer = FeedsMigrateImporter::load('importer_a');
+    $this->assertEquals('importer_a', $importer->id());
+    $this->assertEquals('Importer A', $importer->label());
+    $this->assertEquals(0, $importer->get('importFrequency'));
+    $this->assertEquals('update', $importer->get('existing'));
+    $this->assertEquals(TRUE, $importer->get('keepOrphans'));
+    $this->assertEquals($migration->id(), $importer->get('migrationId'));
+
+    // Assert importer overrides take precedent over migration configuration.
+    // In this example we've only updated the file location. All other values
+    // should be the same as the original migration configuration.
+    $assert_migration_config = [
+      'source' => [
+        'urls' => ['http://news.com/api/articles'],
+      ],
+      'destination' => [],
+    ];
+    $this->assertEquals($assert_migration_config, $importer->get('migrationConfig'));
+  }
+
+  /**
+   * Tests editing an existing importer.
+   */
+  public function testEditImporter() {
+    // Create a migration entity.
+    $migration = $this->createMigration();
+    $migration->set('source', [
+      'plugin' => 'url',
+      'data_fetcher_plugin' => 'file',
+      'data_parser_plugin' => 'json',
+      'item_selector' => '/items/item',
+    ]);
+    $migration->set('destination', [
+      'plugin' => 'entity:node',
+      'default_bundle' => 'article',
+    ]);
+    $migration->save();
+
+    // Create an importer entity.
+    $importer = $this->createImporter();
+    $importer->set('migrationConfig', [
+      'source' => [
+        'data_fetcher_plugin' => 'http',
+        'urls' => [
+          'https://www.news.co.jp/api/articles',
```

Comment:
As said earlier, I think it's better to use www.example.com instead of real looking urls.